CRM/Core/Form.php

```diff
@@ -1584,7 +1584,7 @@ public static function validateMandatoryFields($fields, $values, &$errors) {
    */
   public function getContactID() {
     $tempID = CRM_Utils_Request::retrieve('cid', 'Positive', $this);
-    if (isset($this->_params) && isset($this->_params['select_contact_id'])) {
+    if (isset($this->_params) && !empty($this->_params['select_contact_id'])) {
```

Comment:
@liedekef Franky i think that is something we can look at however this is the exact replica of the code that is in 4.7 aka master, I haven't yet spotted it generating a notice on our dev box but i agree that it feels strange (thanks for the code review tho)